apps/judicial-system/backend/src/app/modules/police/police.service.ts

```diff
@@ -47,8 +48,26 @@ function getChapter(category?: string): number | undefined {
 export class PoliceService {
   private xRoadPath: string
   private agent: Agent
-
   private throttle = Promise.resolve({} as UploadPoliceCaseFileResponse)
+  private policeCaseFileStructure = z.object({
+    rvMalSkjolMals_ID: z.number(),
+    heitiSkjals: z.string(),
+    malsnumer: z.string(),
+    domsSkjalsFlokkun: z.string(),
+    dagsStofnad: z.string(),
```

Comment:
Please make domsSkjalaFlokkun and dagsStofnad optional. I have seen an example where the former is not included and I suspect the latter may also be excluded in some cases.